src/Diagrams/Backend/Miso.hs

```diff
@@ -140,7 +140,7 @@ mouseEventDecoder :: Decoder (Int, Int)
 mouseEventDecoder =
   Decoder
     (withObject "event" $ \o -> liftA2 (,) (o .: "clientX") (o .: "clientY"))
-    mempty
+    (DecodeTarget [])
```

Comment:
Could we please keep this PR to just this change and perhaps bumping some upper bounds (but not removing lower bounds)? I don’t want to support a very old version of GHCJS and ime outdated stack files in repos (and since I don’t use it, it is going to be outdated at some point) are more annoying than helpful.